benches/benches/wgpu-benchmark/shader.rs

```diff
@@ -15,12 +15,11 @@ struct Inputs {
 }
 
 impl Inputs {
-    fn from_dir(folder: &str, extension: &str) -> Self {
+    fn from_dir(folder: &str, extension: &str) -> Result<Self, std::io::Error> {
```

Comment:
This doesn't really feel like a win?

Reply:
It means that the following `unwrap()`'s panic location will be *where the actual path string is defined*, which is useful for making updates like in this PR.

If the `Result` is too much, how about `#[track_caller]`?

Reply:
Actually, I should just back this out because it's better for this PR to be purer moves.

Reply:
Agree on backing out, I think track_caller would be fine.

Reply:
Done; now the only change is adding `#[track_caller]`.